Datasets/data-connect-dataset-teamsconversationactivity.md

```diff
@@ -0,0 +1,106 @@
+| ItemType | string | The type of item being acted on. | No | None |
+| AppName | string | The end user client name being used to generate the signal, E.g. "Unknown", "OutlookMobile", "OWA", "OutlookDesktop", etc. | No | None |
+| AadTenantId | string | AadTenantId of the actor (signal generator). | No | None |
+| ThreadType | string | The type of thread where the conversation is taking place. Among the properties defined by the signal producer. | No |  None |   
+| User_TenantRole | string | Type of user in the tenant (Regular/Admin). Among the properties defined by the signal producer. | No |  None |
+| ThreadName | string | The Channel’s Name. Among the properties defined by the signal producer. | No | None |                    
+| Team_Id | string | Team id, can be chat(for teams chat) or a skype thread id. Among the properties defined by the signal producer. | No |  None |            
+| TeamGroupId | string | Unique GroupId of the teams channel. Among the properties defined by the signal producer. | No | None |                
+| TeamName | string | The team name owning the Channel. Among the properties defined by the signal producer. | No | None |                
+| Emotion | string | Emotion in case the signal is ReactedWithEmoji. Among the properties defined by the signal producer. | No |  None |
+| MembersAdded | string | A list of members added (in case the signal was AddToChannel or Join Channel). Among the properties defined by the signal producer. | No | None |               
+| MembersRemoved | string | A list of members removed (in case the signal was RemoveFromChannel or LeaveChannel). Among the properties defined by the signal producer. | No | None |        
+| ThreadMembers | string | A list of members in the channel (in case the signal was RenameChannel). Among the properties defined by the signal producer. | No | None |        
+| Mentions | string | Mentions in a conversation. Among the properties defined by the signal producer. | No | None |             
+| InteractionMetadata | string | The interaction metadata in a teams conversation. Among the properties defined by the signal producer. | No | None |
+
+
+## JSON representation
+
+```json
+{
+"SignalType": "string", 
+"Id": "string", 
+"StartTime": "datetime", 
+"CreationTime": "datetime", 
+"EndTime": "datetime", 
+"ItemType": "string", 
+"AppName": "string", 
+"AadTenantId": "string", 
+"ThreadType": "string", 
+"User_TenantRole": "string", 
+"ThreadName": "string", 
+"Team_Id": "string", 
+"TeamGroupId": "string", 
+"TeamName": "string", 
+"Emotion": "string", 
+"MembersAdded": "string", 
+"MembersRemoved": "string", 
+"ThreadMembers": "string", 
+"Mentions": "string", 
+"InteractionMetadata": "string", 
+"puser": "String (identifier)", 
+"ptenant": "String (identifier)"
+}
+```
+
+## Sample
+
+```json
```

Comment:
Is it possible to add some sample values in some rows for Team_Id, TeamName, Emotion, MembersAdded, MembersRemoved?